src/components/MigrationModal/PoolInfo/index.tsx

```diff
@@ -108,7 +108,7 @@ const PoolInfo = ({
   }
 
   const unClaimedRow = {
-    label: `${t('migratePage.unclaimedPng')}`,
+    label: `${t('migratePage.unclaimedPng', { pngSymbol: CHAINS[chainId].png_symbol! })}`,
```

Comment:
instead of repeating this `CHAINS[chainId].png_symbol!` everywhere, please create helper function `getPngSymbol` which you can reuse